web/src/components/datasets/DatasetInfo.tsx

```diff
@@ -63,7 +72,7 @@ const DatasetInfo: FunctionComponent<DatasetInfoProps> = (props) => {
   const i18next = require('i18next')
 
   useEffect(() => {
-    run && fetchJobFacets(run.id)
+    run && fetchJobFacets(run.id) && fetchTags()
   }, [])
```

Comment:
I think you would need to add `[run]` to the dependencies list. An empty list runs on the component mount. I think it should work for you.

Reply:
yeah sorry misunderstood what you were saying - rookie error!